src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -66,16 +66,183 @@ public bool GetBooleanValue()
+        /// is written in scientific notation) or, it represents a number less than <see cref="int.MinValue"/> or greater 
+        /// than <see cref="int.MaxValue"/>.
+        /// </exception>
+        public int GetInt32()
+        {
+            if (!TryGetInt32(out int value))
+            {
+                throw ThrowHelper.GetFormatException(NumericType.Int32);
+            }
+            return value;
+        }
+
+        /// <summary>
+        /// Reads the next JSON token value from the source and parses it to a <see cref="long"/>.
+        /// Returns the value if the entire UTF-8 encoded token value can be successfully parsed to a <see cref="long"/>
+        /// value.
+        /// Throws exceptions otherwise.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of JSON token that is not a <see cref="JsonTokenType.Number"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        /// <exception cref="FormatException">
+        /// Thrown if the JSON token value is either of incorrect numeric format (for example if it contains a decimal or 
+        /// is written in scientific notation) or, it represents a number less than <see cref="long.MinValue"/> or greater 
+        /// than <see cref="long.MaxValue"/>.
+        /// </exception>
+        public long GetInt64()
+        {
+            if (!TryGetInt64(out long value))
+            {
+                throw ThrowHelper.GetFormatException(NumericType.Int64);
+            }
+            return value;
+        }
+
+        /// <summary>
+        /// Reads the next JSON token value from the source and parses it to an <see cref="uint"/>.
+        /// Returns the value if the entire UTF-8 encoded token value can be successfully parsed to an <see cref="uint"/>
+        /// value.
+        /// Throws exceptions otherwise.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of JSON token that is not a <see cref="JsonTokenType.Number"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        /// <exception cref="FormatException">
+        /// Thrown if the JSON token value is either of incorrect numeric format (for example if it contains a decimal or 
+        /// is written in scientific notation) or, it represents a number less than <see cref="uint.MinValue"/> or greater 
+        /// than <see cref="uint.MaxValue"/>.
+        /// </exception>
+        [System.CLSCompliantAttribute(false)]
+        public uint GetUInt32()
+        {
+            if (!TryGetUInt32(out uint value))
+            {
+                throw ThrowHelper.GetFormatException(NumericType.UInt32);
+            }
+            return value;
+        }
+
+        /// <summary>
+        /// Reads the next JSON token value from the source and parses it to a <see cref="ulong"/>.
```

Comment:
nit: `...to a <see cref="ulong"/>`

(here and below)

Reply:
hmmm... it is already `to a <see cref="ulong"/>` in those places... Sorry, I didn't get.

Reply:
I meant, `...to an <see cref="ulong"/>` (**an** instead of **a**)